src/engines/html5/capabilities/html5-autoplay.js

```diff
@@ -15,12 +22,35 @@ export default class Html5AutoPlayCapability implements ICapability {
   static runCapability(): void {
     try {
       Html5AutoPlayCapability._vid.src = EncodingSources.Base64Mp4Source;
-      Html5AutoPlayCapability._promise = Html5AutoPlayCapability._vid.play() || Promise.resolve();
+      Html5AutoPlayCapability._promise = Html5AutoPlayCapability._vid.play() || Html5AutoPlayCapability._oldBrowsersCapability();
     } catch (e) {
       Html5AutoPlayCapability._promise = Promise.reject();
     }
   }
 
+  /***
+   * Runs the test for autoplay capability in older browsers, where play() does not return a promise
+   * @static
+   * @returns {Promise<*>} - return a resolve/reject upon the success of playing
+   */
+  static _oldBrowsersCapability(): Promise<*> {
```

Comment:
Why do we need this check? please explain . 
Anyway this is not capability is not accurate

Reply:
Old browsers (such as safari 9) does not return a Promise when activating play(), it returns 'undefined'. In this case (autoplay), it does nothing (and not throwing an error or something..)

So the only way to see that the video start playing is with timeout (any other idea?)